tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -101,7 +99,24 @@
 		<antcall target="dist" inheritall="true" inheritrefs="true" />
 		<antcall target="test-dist-long" inheritall="true" inheritrefs="true" />
 	</target>
-	
+
+	<!-- Runs JavaCC to generate the TLA+ parser files. -->
+	<target name="generate">
+		<tempfile property="javacc-tmpdir" destdir="${java.io.tmpdir}" prefix="javacc_" />
+		<mkdir dir="${javacc-tmpdir}" />
+		<java classpath="lib/javacc-4.0.jar" classname="javacc" fork="true" failonerror="true">
```

Comment:
Have you considered using https://ant.apache.org/manual/Tasks/javacc.html

Reply:
I think that's only for later javacc versions

Reply:
but I will test it!

Reply:
Oh nevermind it wouldn't really work with how we have set up the vendored libs directory.